docs/user-guide/deployment-scenarios.md

```diff
@@ -0,0 +1,51 @@
+# Deployment Scenarios
+
+The StarterKit support a wide variety of deployments, and this article will
+highlight some of them.
+
+- [Keep data on-premise](#keep-data-on-premise)
+- [Redundancy with multiple concentrators and gateways](#redundancy-with-multiple-concentrators-and-gateways)
+- [Deployment on Windows Server](#deployment-on-windows-server)
+- [Deployment of IoTEdge in the cloud](#deployment-of-iotedge-in-the-cloud)
+
+Other scenarios are supported, and combinations of the below scenario are possible.
+
+## Keep data on-premise
+
+It is possible to deploy Azure IoT Edge on your own hardware, and keep all data
+in your local network or infrastructure. For example, by using a custom local
+forwarding module, it is possible to route the data to a local queue or message
+bus. In this case, the sensor data will never leave the on-premise network, and
+connection to Azure will only be used for managing the devices, or handling the
+[deduplication](../adr/007_message_deduplication.md).
+
+![Keep data on-premise](../images/scenarios/scenario-local-data.png)
+
+## Redundancy with multiple concentrators and gateways
+
+In a setup where all sensors can reach at least one concentrator, it is possible
+to get redundancy buy using at lest two concentrators per gateway, and at least
+two gateways. In this case, the deduplication of the messages will happen in two
```

Comment:
[nit] I think it would be more accurate to say that deduplication happens on two levels. The Function will only return whether a message is a duplicate, but it will not actually deduplicate the messages. This is still done on the gateway.